cloudinit/config/cc_growpart.py

```diff
@@ -280,6 +250,36 @@ def resize(self, diskdev, partnum, partdev, fs):
         return (before, get_size(partdev, fs))
 
 
+def resizer_factory(mode: str, distro: Distro, devices: list) -> Resizer:
```

Comment:
This entire function was moved as-is so to be able to reference the `Resizer` return type